src/components/Scorecards/ScorecardsServicePage/ScorecardsServicePage.tsx

```diff
@@ -112,7 +115,7 @@ export const ScorecardsServicePage = () => {
         </Box>
         <Box alignSelf="center">
           <Link
-            to={cortexScorecardServicePageURL(scorecardId, score.serviceId)}
+            to={cortexScorecardServicePageURL({scorecardId, serviceId: score.serviceId, cortexURL: cortexBaseUrl})}
```

Comment:
Why pass it in here instead of making it a part of URL utils? https://github.com/cortexapps/backstage-plugin/blob/master/src/utils/URLUtils.ts#L27

Reply:
Good q let me try that out.

Reply:
I'm not sure there is a good/supported way to do that, unfortunately. https://backstage.io/docs/conf/reading

Reply:
Sighh... this feels like something that will be a bug in the future if we add other URLs

Reply:
Agreed... but I don't have a good fix right now.

Reply:
I suppose we could write the "getUrl" method as a hook instead of a pure function? Feels strange though...

Reply:
That might be better just to prevent future issues

Reply:
I propose we merge this as-is for now and address this in the future, but I don't feel strongly about it. What do you think?

Reply:
Fine with me